### PR TITLE
feat(cce): support storage in node pool

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/cce' TESTARGS='-run TestAccCCENodePool_storage'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cce -v -run TestAccCCENodePool_storage -timeout 360m -parallel 4
=== RUN   TestAccCCENodePool_storage
=== PAUSE TestAccCCENodePool_storage
=== CONT  TestAccCCENodePool_storage
--- PASS: TestAccCCENodePool_storage (906.18s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cce       906.298s
```

### CODE REVIEW

**Comment by Jason-Zhang9309** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/cce' TESTARGS='-run TestAccCCENodePool_storage'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cce -v -run TestAccCCENodePool_storage -timeout 360m -parallel 4
=== RUN   TestAccCCENodePool_storage
=== PAUSE TestAccCCENodePool_storage
=== CONT  TestAccCCENodePool_storage
--- PASS: TestAccCCENodePool_storage (926.08s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cce       926.209s
```

File: docs/resources/cce_node_pool.md [hunk 1]
```diff
@@ -41,6 +41,95 @@ resource "huaweicloud_cce_node_pool" "node_pool" {
...
+    volumetype = "SSD"
+    kms_key_id = var.kms_key_id
+  }
+
+  storage {
+    selectors {
+      name              = "cceUse"
+      type              = "evs"
+      match_label_size  = "100"
+      match_label_count = 1
```
Comment (houpeng80) [inline docs/resources/cce_node_pool.md]: the value type should be string

File: huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go [hunk 2]
```diff
@@ -711,3 +739,94 @@ resource "huaweicloud_cce_node_pool" "test" {
...
+    volumetype = "SSD"
+    kms_key_id = huaweicloud_kms_key.test.id
+  }
+
+  storage {
+    selectors {
+      name              = "cceUse"
+      type              = "evs"
+      match_label_size  = "100"
+      match_label_count = 1
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go]: ditto

File: huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go [hunk 3]
```diff
@@ -181,6 +181,128 @@ func ResourceCCENodePool() *schema.Resource {
...
+									"name": {
+										Type:     schema.TypeString,
+										Required: true,
+										ForceNew: true,
+									},
+									"type": {
+										Type:     schema.TypeString,
+										Optional: true,
+										ForceNew: true,
+										Default:  "evs",
```
Comment (houpeng80) [inline huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go:207]: remove the default value?

Comment (Jason-Zhang9309) [reply huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go:207]: this is a required parameter, but only evs supported, so use a default value is good here

File: docs/resources/cce_node_pool.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/cce_node_pool.md]: fixed

File: huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go]: fixed

**Pull request merged**